docs/sdks/install-sdks.md

```diff
@@ -33,22 +43,34 @@ npm run dist
 </TabItem>
 <TabItem value="py" label="Python">
 
+:::note
+pyC8 requires Python 3.5+. Python 3.6 or higher is recommended.
```

Comment:
Python 3.4 is the minimum supported version of this library we support higher even 3.10